repose-aggregator/tests/release-verification/src/config/scripting.cfg.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<scripting xmlns="http://docs.openrepose.org/repose/scripting/v1.0"
+           language="groovy">
+    if (request.getHeader('X-Auth-Token') == null) {
+        request.addHeader('X-Auth-Token', '12345678901234567890')
+    }
+</scripting>
```

Comment:
The logged Origin service hit was from the test during the Vagrant up, not from the Scripting filter bypassing the chain. So as previously thought, the Scripting filter was bypassing not just the chain, but also the Origin and returning directly with an OK (200).